docs/pages/api.rst

```diff
@@ -965,14 +1019,19 @@ Signature: ``Collection::ifThenElse(callable $condition, callable $then, ?callab
 implode
 ~~~~~~~
 
-Convert all the elements of the collection to a single string.
+Join all the elements of the collection into a single string using a glue provided or the empty string as default.
 
-The glue character can be provided, default is the empty character.
+Note internally ``foldLeft`` is used, which is why the result will have the last element's key.
```

Comment:
Nice! Should we add a tips box for this?

Reply:
Yeah that sounds good, will add 👍